invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -109,10 +109,17 @@ <h1 class="ui large header mb-0">{{ community.metadata.title }}</h1>
     <div
       class="ui container secondary pointing stackable menu page-subheader pl-0 pr-0">
       {% for item in current_menu.submenu('communities').children if item.permissions == True or permissions[item.permissions] %}
-        <a
-          class="item {{ 'active' if active_community_header_menu_item == item.name }} {{ 'disabled' if not item.url }}"
-          href="{{ item.url }}"><i aria-hidden="true"
-                                   class="{{ item.icon }} icon"></i>{{ item.text }}</a>
+        {% if item.name == "curation_policy" and not community.metadata.curation_policy %}
```

Comment:
Empty `if` content? Then, we can rework the conditions to avoid that?

Reply:
Empty if no content :) `if name and not content`